### PR TITLE
[SR] Add quality settings

### PR DESCRIPTION
**Pull request by romtsn**
_#skip-changelog_

## :scroll: Description

* I've picked a bit higher values for bps than iOS, because I've tested it and somehow 40kbps for medium looks pretty bad, especially on compose screens

## :bulb: Motivation and Context

Closes https://github.com/getsentry/team-replay/issues/444

### CODE REVIEW

File: sentry/src/main/java/io/sentry/SentryReplayOptions.java [hunk 1]
```diff
@@ -39,10 +51,10 @@ public final class SentryReplayOptions {
   private boolean redactAllImages = true;
 
   /**
-   * Defines the quality of the session replay. Higher bit rates have better replay quality, but
-   * also affect the final payload size to transfer, defaults to 100kbps.
+   * Defines the quality of the session replay. The higher the quality, the more accurate the replay
+   * will be, but also more data to transfer and more CPU load, defaults to MEDIUM.
    */
-  private int bitRate = 100_000;
+  private SentryReplayQuality quality = SentryReplayQuality.MEDIUM;
```
Comment (romtsn) [inline sentry/src/main/java/io/sentry/SentryReplayOptions.java:57]: @brustolin we said in the [ticket](https://github.com/getsentry/team-replay/issues/444) that the default should be `medium`, but I see you [use](https://github.com/getsentry/sentry-cocoa/blob/cd76c54171981b2f53636e9ee379725c9a3188b5/Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift#L67) `low` - do you wanna change that, or should we re-phrase the ticket?

File: sentry/src/test/java/io/sentry/SentryReplayOptionsTest.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import kotlin.test.assertEquals
+
+class SentryReplayOptionsTest {
+
+    @Test
+    fun `uses medium quality as default`() {
+        val replayOptions = SentryReplayOptions()
+
+        assertEquals(SentryReplayOptions.SentryReplayQuality.MEDIUM, replayOptions.quality)
+        assertEquals(40_000, replayOptions.bitRate)
```
Comment (stefanosiano) [inline sentry/src/test/java/io/sentry/SentryReplayOptionsTest.kt:13]: guess you have to update these values, yet

File: sentry/src/main/java/io/sentry/SentryReplayOptions.java [hunk 3]
```diff
@@ -121,9 +133,27 @@ public void setRedactAllImages(final boolean redactAllImages) {
...
+  }
+
+  /**
+   * Defines the quality of the session replay. Higher bit rates have better replay quality, but
+   * also affect the final payload size to transfer, defaults to 40kbps.
+   */
   @ApiStatus.Internal
   public int getBitRate() {
-    return bitRate;
+    return quality.ordinal() * 25_000 + 50_000;
```
Comment (stefanosiano) [inline sentry/src/main/java/io/sentry/SentryReplayOptions.java:150]: `s`: Can you merge bitRate and sizeScale inside the `SentryReplayQuality` enum?

**Review by stefanosiano — APPROVED** [review]

File: sentry/src/main/java/io/sentry/SentryReplayOptions.java (hunk 1 above)
Comment (brustolin) [reply sentry/src/main/java/io/sentry/SentryReplayOptions.java:57]: Lets use medium, no problem.

**Pull request merged**